### ISSUE TITLE
Baritone lily pad pakour

### ISSUE THREAD

**Issue by Schweeeeeeeeeeeeeeee** [cabaletta/baritone#2964]
baritone already lands on lily pads and walks on them but preffers to swimm through water than pakour over them

just for faster travel in swamps 

## Final checklist

- [x] I know how to properly use check boxes 
- [x] I have not used any OwO's or UwU's in this issue.

**Comment by cazeip** [comment]
I agree, even with `allowParkour` turned on, Baritone does not pathfind on top of lilypads.
